### PR TITLE
test: multiple organizer address test

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

add test case for multiple organizer address feature

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Chore (refactoring code, technical debt, workflow improvements)

### CODE REVIEW

**Review by emrysal — DISMISSED** [review]
The test is failing 😅

File: apps/web/components/ui/form/LocationSelect.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ export default function LocationSelect(props: Props<LocationOption, false, Group
   return (
     <Select<LocationOption>
       name="location"
+      id="location-select"
```
Comment (hariombalhara) [inline apps/web/components/ui/form/LocationSelect.tsx:43]: Does `data-testid` not work? We should use a test-specific attribute to target elements in tests.

Comment (G3root) [reply apps/web/components/ui/form/LocationSelect.tsx:43]: tried using `data-testid` but it's not passed on to the container element.

Comment (hariombalhara) [reply apps/web/components/ui/form/LocationSelect.tsx:43]: Cool. It;s fine then I guess.

**Comment by hariombalhara** [comment]
Thanks @G3root for your first contribution to tests!! Many to come 🙏

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**